New log file opened at Sun Aug 14 11:29:19 BRT 2016 by 4186@papejajunior-NH4CU03
Taking next connection to: 35839
New client attempting to connect on port: 50071-35839
  Address = /127.0.1.1:35839
This client on port 50071 has uuid 7629ea20-ee1b-48d1-8d04-a5112c916188 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-7629ea20-ee1b-48d1-8d04-a5112c916188
Taking next connection to: 35839
New client attempting to connect on port: 50072-35839
  Address = /127.0.1.1:35839
Handshake exception on socket: 50072
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 35839
New client attempting to connect on port: 50073-35839
  Address = /127.0.1.1:35839
This client on port 50073 has uuid d67c0076-4363-4118-9829-aee30a620a58 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-7629ea20-ee1b-48d1-8d04-a5112c916188, activator-d67c0076-4363-4118-9829-aee30a620a58
Taking next connection to: 35839
New client attempting to connect on port: 50076-35839
  Address = /127.0.1.1:35839
Handshake exception on socket: 50076
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 35839
New client attempting to connect on port: 50077-35839
  Address = /127.0.1.1:35839
This client on port 50077 has uuid 036579ed-9365-4d39-9fa5-de56110ef3a4 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-d67c0076-4363-4118-9829-aee30a620a58, play-fork-run-036579ed-9365-4d39-9fa5-de56110ef3a4
Taking next connection to: 35839
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 35839
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 35839
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 35839
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 35839
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 35839
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 35839
New client attempting to connect on port: 50212-35839
  Address = /127.0.1.1:35839
Handshake exception on socket: 50212
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 35839
New client attempting to connect on port: 50213-35839
  Address = /127.0.1.1:35839
This client on port 50213 has uuid 84d301b8-0345-4c1b-a490-d6caa6913f83 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-d67c0076-4363-4118-9829-aee30a620a58, play-fork-run-84d301b8-0345-4c1b-a490-d6caa6913f83
Taking next connection to: 35839
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 35839
